### DISCUSSION TITLE
Failed to run `config` for nvim-lspconfig

### DISCUSSION THREAD

**Discussion by 3ynm** [LazyVim/LazyVim#2896] [Q&A] [answered]
After an upgrade I'm getting the following error:

```
Failed to run `config` for nvim-lspconfig

...share/nvim/lazy/LazyVim/lua/lazyvim/plugins/lsp/init.lua:123: attempt to index field 'signs' (a boolean value)

# stacktrace:
  - /LazyVim/lua/lazyvim/plugins/lsp/init.lua:123 _in_ **config**
  - /LazyVim/lua/lazyvim/util/plugin.lua:92 _in_ ****
  - vim/_editor.lua:0
```

Is this a bug or I need to change something on my config?

**Comment by dpetka2001** [comment]
Have you disabled `diagnostics.signs` in your configuration?

**Reply by 3ynm** [reply]
Yes.

```
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      diagnostics = {
        virtual_text = false,
        signs = false,
      },
    },
  },
}
```

**Reply by dpetka2001** [reply, marked as answer]
I submitted #2897 that will hopefully fix this.

**Reply by dpetka2001** [reply]
If you want to try it until it gets fixed, you can do 
```
{
  "dpetka2001/LazyVim",
  branch = "fix/lsp-signs",
}
```
in the meantime. 
Make sure to update from Lazy UI if you decide to do this. And after it gets merged, you can delete the above snippet of code and update again from the Lazy UI. Or you could comment the line `signs = false` in your configuration, so that you don't get the error, until it gets merged.
